lib/ExpressionUtils/ExpressionUtilsImpl.py

```diff
@@ -45,8 +39,8 @@ class ExpressionUtils:
     # the latter method is running.
     ######################################### noqa
     VERSION = "0.1.1"
```

Comment:
Please bump the version numbers

Reply:
Make sure you update the kbase.yml file as well

Reply:
version number to 0.1.2 I suppose?  Will do

There's no UI entry for this new method - I didn't add one because it was only to be invoked by the javascript code of Jim.  Was that an error on my part?

Reply:
Oh you mean version in the yaml file - I get it.   Will do.